cmake/SetupDoxygen.cmake

```diff
@@ -14,17 +14,23 @@ if (DOXYGEN_FOUND)
   # Configure file that contains doxygen groups
   configure_file(docs/GroupDefs.hpp ${SPECTRE_DOXYGEN_GROUPS})
 
+  # Construct the command that calls Doxygen
   set(
     GENERATE_DOCS_COMMAND
-    ${DOXYGEN_EXECUTABLE}
-    ${PROJECT_BINARY_DIR}/docs/DoxyfileHtml
+    "${DOXYGEN_EXECUTABLE} ${PROJECT_BINARY_DIR}/docs/DoxyfileHtml"
```

Comment:
Why is this changing? I had it on separate lines so that if more arguments are added the diff is less likely to create conflicts.

Reply:
I had to make this a string (as opposed to a CMake list) to assemble the command in CMake. That's why it's in one line now.